gcovr/tests/nobranch/Makefile

```diff
@@ -0,0 +1,21 @@
+all:
+	$(CXX) -fprofile-arcs -ftest-coverage -fPIC main.cpp -o testcase
+
+run: txt xml html
+
+txt:
+	./testcase
+	../../../scripts/gcovr -d --fail-under-branch 100.0 -o coverage.txt
```

Comment:
Would it make sense to use the `--branch` report here, to test that it too can deal sensibly with 0/0 branch coverage?

Reply:
done. It makes perfect sense.